terraform/datadog/concourse-jobs.tf

```diff
@@ -11,4 +11,13 @@ resource "datadog_timeboard" "concourse-jobs" {
        q = "${format("avg:%s_concourse.build.finished{*} by {job}", replace(var.env, "-", "_"))}"
     }
   }
+
+  graph {
+    title = "CF pipeline run time"
+    viz = "timeseries"
+    request {
+      q = "${format("avg:%s.pipeline_time{environment:%s}", replace(var.env, "-", "_"), replace(var.env, "-", "_"))}"
```

Comment:
we no longer need to replace - with _, this is changed in https://github.com/alphagov/paas-cf/pull/522/commits/12c25b31191b21e6b704988934a7780e5ff33dcf